cli/asc.js

```diff
@@ -592,7 +616,9 @@ exports.main = function main(argv, options, callback) {
       for (let j = 0, l = libFiles.length; j < l; ++j) {
         let libPath = libFiles[j];
         let libText = readFile(libPath, libDir);
-        if (libText === null) return callback(Error("Library file '" + libPath + "' not found."));
+        if (libText === null) {
```

Comment:
Any reason for not `!libText`?

Reply:
I've seen it other places too, just wondering why not just `falsy`?

Reply:
Hmm, yeah. It could be simplify

Reply:
We can't do this due to `!libText` and `libText == null` is not the same thing. `!libText` means it could be non-null but with zero-lenght. So it exists but empty (which is valid in general case)